### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by svwk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # progr_eng-II
 Software engineering project. 2 sem. 
+sdsgdgd
```
Comment (Dmitrii-Krasnov) [inline README.md:3]: strange symbols

**Review by Dmitrii-Krasnov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (svwk) [reply README.md:3]: may be. I just like it

**Review by GLOBB1000 — CHANGES_REQUESTED** [review]
Loop

File: README.md (hunk 1 above)
Comment (GLOBB1000) [reply README.md:3]: Loop

**Pull request closed without merging**